src/Silex/Provider/SwiftmailerServiceProvider.php

```diff
@@ -91,26 +91,30 @@ public function register(Container $app)
 
             $plugins = $app['swiftmailer.plugins'];
 
+            $keys = $plugins->keys();
+            sort($keys);
```

Comment:
> I don't see how sorting implicitly works towards more predictability.

I imagined something like this, similar to [conf.d](http://superuser.com/questions/705297/in-what-order-does-apache-load-conf-files-and-which-ones/705317#705317):
```
$app['swiftmailer.plugins']['001-bcc] = ... plugin that should run first ...
$app['swiftmailer.plugins']['002-filter'] = ... plugin that should run later ...
$app['swiftmailer.plugins']['foo'] = ... plugin where order does not matter  ...
```
The use-case for this is if multiple providers register plugins where the plugin order is important. If you register multiple plugins in your own code within a single function, I agree that keeping the registration order is more intuitive.

Anyway, I don't have strong feelings about this, so I will remove it.